test/Feature/QueryLimitedSpec.hs

```diff
@@ -15,59 +15,56 @@ spec :: SpecWith ((), Application)
 spec =
   describe "Requesting many items with server limits(max-rows) enabled" $ do
     it "restricts results" $
-      get "/items"
-        `shouldRespondWith` [json| [{"id":1},{"id":2}] |]
-        { matchStatus  = 200
-        , matchHeaders = ["Content-Range" <:> "0-1/*"]
-        }
+      get "/items?order=id"
```

Comment:
Rolled back updates are not entirely without consequence. They change the order of rows on disk. That means that requesting `/items` will return those in a different order. Since PG does not give any guarantee about the order of results without `ORDER BY`, this is something that should be changed in any case. Those that were breaking, I added a `?order=` part.

Reply:
Agree, I had to do a similar change before.